frontend/micro-ui/web/micro-ui-internals/packages/Mukta/src/configs/UICustomizations.js

```diff
@@ -481,16 +481,18 @@ export const UICustomizations = {
   },
   SearchProjectConfig: {
     preProcess: (data) => {
```

Comment:
@Ramkrishna-egov  / @nipunarora-eGov  can you create a different config for WMS Search 
in this UI Customisation file
since we have to make the change if we ever back to plain search